cmd/pruner.go

```diff
@@ -33,11 +31,12 @@ func setConfig(cfg *log.Config) {
 	cfg.Level = zerolog.InfoLevel
 }
 func PruneAppState(dataDir string) error {
-	o := opt.Options{
-		DisableSeeksCompaction: true,
+	backend, err := GetFormat(filepath.Join(dataDir, "state.db"))
+	if err != nil {
+		return err
 	}
 
-	appDB, err := db.NewGoLevelDBWithOpts("application", dataDir, &o)
+	appDB, err := db.NewDB("application", backend, dataDir)
```

Comment:
interesting. Just couple of thoughts

Took me couple of minutes to understand that `db` comes from `db "github.com/cosmos/cosmos-db"` and not the `db.go`, but that's just Golang's module system throwing me off

also, was surprized that the interface is the same, but then realised that it's not the original https://github.com/cometbft/cometbft-db